connectors/aws/aws-lambda/element-templates/aws-lambda-connector.json

```diff
@@ -39,7 +44,7 @@
   "properties": [
     {
       "type": "Hidden",
-      "value": "io.camunda:aws-lambda:1",
+      "value": "io.camunda:aws:lambda:1",
```

Comment:
@chillleader I thought the templates are versioned using their `version` property and not based on the properties themselves?

Reply:
The template itself will be fine, and the user will be able to update to a newer version seamlessly, so that's ok. But since we use this property in the runtime to select the connector function to run, it will break when the execution gets to connector.

Let's say someone still has the old connector runtime in the cluster, that only knows a connector called `io.camunda:aws-lambda:1`. User has updated the diagram to use the new template which is now called `io.camunda:aws:lambda:1`. The runtime will fail to resolve the connector function with this type.